packages/database/e2e/helpers.js

```diff
@@ -1,10 +1,10 @@
-const testingUtils = require('@firebase/rules-unit-testing');
+// const testingUtils = require('@firebase/rules-unit-testing');
```

Comment:
Source control allows us to be bold - don't leave commented out code, if you've got something working, delete the lines

I love diffs that remove more lines than they add ;-)

Reply:
Sorry, this slipped through the cracks while I was rebasing to edit the commits with the right author for the CLA.